instrumentation/opentelemetry-instrumentation-httpx/tests/test_httpx_integration.py

```diff
@@ -741,8 +743,10 @@ def create_proxy_transport(self, url: str):
 
         def setUp(self):
             super().setUp()
-            HTTPXClientInstrumentor().instrument()
             self.client = self.create_client()
+            HTTPXClientInstrumentor().instrument_client(self.client)
+
+        def tearDown(self):
```

Comment:
This is the change of behavior, before uninstrument() did not touch already instrumented clients

Reply:
I believe this is fine.